metadome/presentation/web/templates/dashboard.html

```diff
@@ -3,7 +3,7 @@
 <!-- Scripts needed for functionality on this page -->
 <script src="{{ url_for('static', filename='js/d3-tip.js') }}"></script>
 <script src="{{ url_for('static', filename='js/dashboard/visualization.js') }}" async></script>
-<script src="{{ url_for('web.dashboard_js') }}" async></script>
+<script src="{{ url_for('web.dashboard_js') }}" type="text/javascript"></script>
```

Comment:
Why did you remove the async here?

Reply:
The javascript on the page needs to call the function "visualizeTranscript", which is located in dashboard.js. If the script is called asynchronously, then the page wont have the definition of the function and the call will fail. For that reason, I made sure that dashboard.js is called before the rest of the page loads.

Reply:
Okay... I explicitly made this async previously because of some Jinja constructions in the javascript would not work otherwise. I now am not running into any problems though...

2 questions
Have you experienced JINJA not working wthout async before?
Is the manner of 'url_for' related to this. e.g.: https://github.com/cmbi/metadome/blob/transcript-url/metadome/presentation/web/templates/js/dashboard.js#L200 
vs https://github.com/cmbi/metadome/blob/transcript-url/metadome/presentation/web/templates/js/dashboard.js#L373

Reply:
I managed to make the javascript in dashboard.js use the url of the page, to extract the transcript id from. I moved the call in https://github.com/cmbi/metadome/pull/57/commits/3e6fe0cebc6db34994274e8bdc3d7ecddc1eaa96. I don't know which solution is preferred.

I've never experienced JINJA not working without async. I don't think the two are related.
That JSGlue call to 'url_for' should work normally. Is the problem still ocurring?

Reply:
Seems to work perfectly thanks Coos